.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
   matrix:
     - "TESTLANG=C"
     - "TESTLANG=CSharp"
+    - 'TESTDIR="C/lwan"'
```

Comment:
You can remove this line. The framework will be tested with `"TESTLANG=C"`

Reply:
Done.